setup.py

```diff
@@ -86,6 +86,7 @@
 					'any = web.security.predicate:any',
 					'matches = web.security.predicate:ContextMatch',
 					'contains = web.security.predicate:ContextContains',
+					'csrf_passes = web.security.csrf:csrf_predicate',
```

Comment:
So I didn't do anything wrong here? For whatever reason when.csrf_passes says it cannot be found.